iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -50,7 +50,12 @@ public enum ErrorCode
         // Note: one example found in logs is for invalid characters in a twin property name.
         ArgumentInvalid = 400004,
 
+        /// <summary>
+        /// Something in the payload is unexpectedly null. Check the error message for more information about what is invalid.
+        /// </summary>
+        // Note: an example suggest is null method payloads, but our client converts null to a JSON null, which is allowed.
```

Comment:
suggested?

Reply:
To use the cool trick James taught us:

```suggestion
        // Note: an example suggested is null method payloads, but our client converts null to a JSON null, which is allowed.
```